templates/zerver/help/read-receipts.md

```diff
@@ -0,0 +1,83 @@
+# Read receipts
+
+Read receipts let you check who has read a message. You can see read receipts
+for any message, including both [stream messages](/help/streams-and-topics) and
+[private messages](/help/private-messages).
+
+With privacy in mind, Zulip lets you [control][configure-personal-read-recipts]
+whether your read receipts are shared, and administrators can
+[choose][configure-organization-read-recipts] whether to enable read receipts in
+their organization.
+
+!!! tip ""
+    Read receipts reflect whether or not someone has marked a message as read,
+    whether by viewing it, or [by marking messages as read in
+    bulk](/help/marking-messages-as-read).
+
+## View who has read a message
+
+{start_tabs}
+
+{!message-actions-menu.md!}
+
+3. Click **View read receipts**.
+
+!!! tip ""
+    You will also see how many people have read the message.
+
+{end_tabs}
+
+## Configure whether Zulip lets others see when you've read messages
+
+Zulip supports the privacy option of never sharing whether or not you have read
+a message.
+
+{start_tabs}
+
+{settings_tab|account-and-privacy}
+
+1. Under **Privacy**, toggle **Let others see when I've read messages**.
+
+{end_tabs}
+
+## Configure read receipts for your organization
+
+{!admin-only.md!}
+
+You can configure:
+
+* Whether read receipts are enabled in your organization.
+* Whether new users will allow others to view read receipts by default. (Note
+  that users [can always change this setting][configure-personal-read-recipts]
+  once they join.)
+
+### Configure whether read receipts are enabled in your organization
+
+{start_tabs}
+
+{settings_tab|organization-settings}
+
+1. Under **Other settings**, toggle **Enable read receipts**.
+
+{!save-changes.md!}
+
+{end_tabs}
+
+### Configure default read receipt sharing settings for new users
+
+{start_tabs}
+
+{settings_tab|default-user-settings}
+
+1. Under **Privacy settings**, toggle **Allow other users to view read receipts**.
+
+{end_tabs}
+
+## Related articles
+
+* [Status and availability](/help/status-and-availability)
+* [Typing notifications](/help/typing-notifications)
+
```

Comment:
Maybe we should also add `/help/streams-and-topics` and `/help/private-messages` to **Related articles** since these help center docs were mentioned in the text?

Reply:
Hm, I think we can leave those off - the links are there in the body as background material, but I don't think it's super likely that someone reading this article specifically needs to refer to those.